modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/upgrade/v4_2_1/test/UpgradeCalendarBookingTest.java

```diff
@@ -119,6 +130,22 @@ protected void assertSameTime(
 			actualJCalendar.get(java.util.Calendar.MINUTE));
 	}
 
+	protected CalendarBooking createCalendarBooking() throws PortalException {
+		setUserTimeZoneId();
+
+		java.util.Calendar expectedStartTimeJCalendar =
```

Comment:
Hi @rijema, should we use the expected prefix here?

Reply:
@rijema I don't think we should use the expected prefix